### PR TITLE
Support copy file name

### LINKED ISSUE #998: Copy file name
**Is your feature request related to a problem? Please describe.**
I use `ctrl+o` a lot for copying branches and commits, but it doesn't work on a file name (e.g. inside of a commit or inside the files panel). Perhaps we could extend that shortcut to work in other contexts that display lists of files?

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Yuuki77**
Fix #998

You can copy the file nae by clicking `ctrl+o` in inside of a commit or inside the files panel.
This is my first pr to this repo 
If you have any comments, please let me know 😄

### CODE REVIEW

File: pkg/i18n/english.go [hunk 1]
```diff
@@ -1149,6 +1149,12 @@ func addEnglish(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "copyBranchNameToClipboard",
 			Other: "copy branch name to clipboard",
+		}, &i18n.Message{
+			ID:    "copyFileNameToClipboard",
+			Other: "Copy the file name to the clipboard",
```
Comment (jesseduffield) [inline pkg/i18n/english.go:1154]:
```suggestion
			Other: "copy the file name to the clipboard",
```
descriptions for keybindings are conventionally in lower-case :)

**Review by jesseduffield — CHANGES_REQUESTED** [review]
Thanks for making this PR! Just one comment about casing of the keybinding descriptions

**Comment by Yuuki77** [comment]
@jesseduffield Thank you for the review! I just updated it 😄

File: pkg/i18n/english.go [hunk 2]
```diff
@@ -1149,6 +1149,12 @@ func addEnglish(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "copyBranchNameToClipboard",
 			Other: "copy branch name to clipboard",
+		}, &i18n.Message{
+			ID:    "copyFileNameToClipboard",
+			Other: "copy the file name to the clipboard",
+		}, &i18n.Message{
+			ID:    "copyCommitFileNameToClipboard",
+			Other: "Copy the committed file name to the clipboard",
```
Comment (jesseduffield) [inline pkg/i18n/english.go:1157]: we'll also want to downcase this

File: pkg/i18n/dutch.go [hunk 3]
```diff
@@ -1144,6 +1144,12 @@ func addDutch(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "copyBranchNameToClipboard",
 			Other: "copieer branch name naar clipboard",
+		}, &i18n.Message{
+			ID:    "copyFileNameToClipboard",
+			Other: "Kopieer de bestandsnaam naar het klembord",
```
Comment (jesseduffield) [inline pkg/i18n/dutch.go:1149]: we'll also want to downcase this

File: pkg/i18n/dutch.go [hunk 4]
```diff
@@ -1144,6 +1144,12 @@ func addDutch(i18nObject *i18n.Bundle) error {
...
+		}, &i18n.Message{
+			ID:    "copyCommitFileNameToClipboard",
+			Other: "Kopieer de vastgelegde bestandsnaam naar het klembord",
```
Comment (jesseduffield) [inline pkg/i18n/dutch.go:1152]: we'll also want to downcase this

**Review by jesseduffield — CHANGES_REQUESTED** [review]
couple more things :)

**Comment by Yuuki77** [comment]
@jesseduffield Sorry I missed them.
It should be good now 💪

**Comment by jesseduffield** [comment]
Tested and it works! thanks heaps for making this, I'll be using it myself :)

**Pull request merged**